From: Phil Morrell <debian@emorrp1.name>
Date: Fri, 11 May 2018 00:50:10 +0100
Subject: add system installation detection

* if installed in /usr, set PLAYIT_LIB2 to system location
* user override provided by specifying env var explicitly
---
 play.it | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/play.it b/play.it
index daf6a9b..27cb360 100755
--- a/play.it
+++ b/play.it
@@ -2,6 +2,7 @@
 
 set -o errexit
 
+DIR=$(readlink --canonicalize "$(dirname "$0")")
 INPUT_FILE="$1"
 shift 1
 
@@ -9,7 +10,9 @@ LIB_ONLY=true
 
 if [ -z "$PLAYIT_LIB2" ]; then
 	[ -n "$XDG_DATA_HOME" ] || XDG_DATA_HOME="$HOME/.local/share"
-	if [ -e "$XDG_DATA_HOME/play.it/libplayit2.sh" ]; then
+	if [ "$(echo "$DIR" | cut --delimiter=/ --fields=2)" = "usr" ]; then
+		PLAYIT_LIB2='/usr/share/games/play.it/libplayit2.sh'
+	elif [ -e "$XDG_DATA_HOME/play.it/libplayit2.sh" ]; then
 		PLAYIT_LIB2="$XDG_DATA_HOME/play.it/libplayit2.sh"
 	elif [ -e './libplayit2.sh' ]; then
 		PLAYIT_LIB2='./libplayit2.sh'
